src/views/Admin/Pages/Index.vue

```diff
@@ -20,6 +20,7 @@
                 </b-input-group>
               </b-form-group>
             </form>
+            <import :namespaceID="namespace.namespaceID" type="page" />
```

Comment:
We use Namespace object everywhere (afaik). any special reason not to use it here?

Reply:
Passing the whole namespace object felt like a lot of unneeded information. For consistency i will change the prop to namespace.